internal/resource/configmap.go

```diff
@@ -99,6 +99,11 @@ func (builder *ServerConfigMapBuilder) Update(object runtime.Object) error {
 				return err
 			}
 		}
+		if builder.Instance.Spec.TLS.DisableNonTLSListeners {
+			if _, err := defaultSection.NewKey("listeners.tcp", "none"); err != nil {
+				return err
+			}
+		}
```

Comment:
As described in https://github.com/rabbitmq/cluster-operator/issues/459#issuecomment-725378817, we also need further settings if Web MQTT or Web STMOP is enabled and `DisableNonTLSListeners` is set.

Reply:
silly me indeed. Mark the PR as draft for now

Reply:
added settings for stomp, mqtt, web_stomp and web_mqtt. For management, the general `listeners.tcp = none` had the effect. Thanks for bring it up David.